src/blueprint/from_config/schema.rs

```diff
@@ -50,9 +70,11 @@ fn validate_mutation(config: &Config) -> Valid<(), String> {
       return Valid::fail("Mutation type is not defined".to_owned()).trace(mutation_type_name);
     };
 
-    Valid::from_iter(mutation.fields.iter(), validate_field_has_resolver)
-      .trace(mutation_type_name)
-      .unit()
+    Valid::from_iter(mutation.fields.iter(), |i| {
+      validate_field_has_resolver(i, &config.types)
+    })
+    .trace(mutation_type_name)
+    .unit()
```

Comment:
it could be replaced like this here? 
```suggestion
    validate_field_of_type_has_resolver(mutation_type_name, mutation, &config.types)
```
and for query as well